src/internal/modules/charge-information/forms/charge-element/quantities.js

```diff
@@ -8,16 +8,18 @@ const { getChargeElementData, getChargeElementActionUrl } = require('../../lib/f
 
 const getErrors = key => {
   const errors = {
-    'string.pattern.base': {
+    'number.unsafe': {
+      message: `Enter a number for the ${key} quantity using 6 decimal places or fewer, the number must be more than 0`
```

Comment:
Perhaps set the message as a constant and reuse for both number.unsafe and number.custom.